lib/main-app.js

```diff
@@ -97,6 +100,22 @@ function spawnFinder () {
   })
 }
 
+function openURL (url) {
+  //verify that url matches boostnote custom url scheme
+  let boostnoteUrlScheme = 'boost';
+  let urlPrefix = boostnoteUrlScheme + '://';
```

Comment:
Please use `const` instead of `let`!
And please use template literal. You can write below.

```javascript
const urlPrefix = `${boostnoteUrlScheme}://`
```